projects/core/src/lib/download-layer/download-layer.component.ts

```diff
@@ -132,6 +135,24 @@ export class DownloadLayerComponent implements OnInit, OnDestroy {
         this.viewportSizeSubscription?.unsubscribe();
     }
 
+    unsubscribeTime(): void {
+        this.projectTimeSubscription?.unsubscribe();
+    }
+
+    setEditedExtent(): void {
+        this.editedExtent = true;
```

Comment:
this should also be set when the user selected a region using "select region on the map"

Reply:
https://github.com/geo-engine/geoengine-ui/pull/734/commits/54a43b93387d478e4c1ae46410c05115db03396b